content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -389,7 +389,7 @@ kubectl cluster-info                                                  # Display
 kubectl cluster-info dump                                             # Dump current cluster state to stdout
 kubectl cluster-info dump --output-directory=/path/to/cluster-state   # Dump current cluster state to /path/to/cluster-state
 
-# View existing taints on which exist on current nodes.
+# "View existing taints on which exist on current nodes."
 kubectl get nodes -o=custom-columns=NodeName:.metadata.name,TaintKey:.spec.taints[*].key,TaintValue:.spec.taints[*].value,TaintEffect:.spec.taints[*].effect
```

Comment:
```suggestion
# View existing taints on which exist on current nodes.
kubectl get nodes -o='custom-columns=NodeName:.metadata.name,TaintKey:.spec.taints[*].key,TaintValue:.spec.taints[*].value,TaintEffect:.spec.taints[*].effect'
```
mmh I guess what you wanted to do to fix https://github.com/kubernetes/website/issues/36640 was quoting the actual command flag parameters.

Reply:
@NitishKumar06 , does the suggested change make sense?

Reply:
@mtardy Thank you for mentioning this!

Reply:
will you apply this change @NitishKumar06 or should I create a separate issue? :)

Reply:
@mtardy Sorry for being late! The changes have been applied now:)

Reply:
Hey, no problem but it doesn't seem that you applied the change, it's still missing the singlequotes, and as tengqm stated, you left a double quote at the end!